tests/components/scrape/__init__.py

```diff
@@ -103,6 +103,13 @@ async def async_update(self, data: bool | None = True) -> None:
                 "<div class='links' style='links'><a href='/latest-release-notes/'>Release notes</a></div></div>"
                 "<template>Trying to get</template>"
             )
+        if self.payload == "test_scrape_sensor2":
```

Comment:
I had difficulty working out the difference with the above...
I think a comment is nice.
```suggestion
        if self.payload == "test_scrape_sensor2":
            # Hidden version
```